doc/source/whatsnew/v0.21.0.txt

```diff
@@ -116,6 +116,8 @@ Other API Changes
 Deprecations
 ~~~~~~~~~~~~
 - :func:`read_excel()` has deprecated ``sheetname`` in favor of ``sheet_name`` for consistency with ``.to_excel()`` (:issue:`10559`).
+- :func:`Series.argmax` has been deprecated in favor of :func:`Series.idxmax` (:issue:`16830`)
```

Comment:
> You can make your own mini-section.

@TomAugspurger : That would work, and I think you already wrote ~90% of it :smile:

My only critique ATM would be that we can just say what the current behavior is (instead of saying how long we've had this aliasing).